docs/GuiTestDocumentation.md

```diff
@@ -0,0 +1,282 @@
+
+#### Edit a friend
+
+Upon successful editing of a friend
+
+1. Correct Command result display message
+2. Friend should be edited and if name is changed, list should be reordered
+
+|                       Before                       |                      After                       |
+|:--------------------------------------------------:|:------------------------------------------------:|
+| ![BeforeEdit](images/GUITestImages/BeforeEdit.png) | ![AfterEdit](images/GUITestImages/AfterEdit.png) |
+
+As you can see in the above images `Naaman` was edited to `Bean` and the list was reordered. The appropriate command result message displayed was also in the CommandResult box.
+
+If incorrect format of edit command is entered, the appropriate edit friend error message should be displayed in the CommandResult box. Shown below is the error message in the CommandResultBox.
+
+![EditError](images/GUITestImages/EditError.png)
+
+#### Deleting a friend
+
+Upon successful deletion of a friend
+
+1. Correct Command result display message
+2. Friend card should be removed from the GUI
+
+|                         Before                         |                          After                           |
+|:------------------------------------------------------:|:--------------------------------------------------------:|
+| ![BeforeDelete](images/GUITestImages/BeforeDelete.png) | ![AfterDelete.png](images/GUITestImages/AfterDelete.png) |
+
+As you can see in the above images `Bean` was deleted with the appropriate command result message displayed in the CommandResult box.
+
+If incorrect format of delete command is entered, the appropriate delete friend error message should be displayed in the CommandResult box. Shown below is the error message in the CommandResultBox.
+
+![DeleteError](images/GUITestImages/DeleteError.png)
+
+#### Show a specific friend
+
+Upon successful execution of `showfriend` command the following must occur
+
+1. Correct Command result display message
+2. Expanded Friend card of friend requested should be displayed
+
+|                             Before                             |                            After                             |
+|:--------------------------------------------------------------:|:------------------------------------------------------------:|
+| ![BeforeShowFriend](images/GUITestImages/BeforeShowFriend.png) | ![AfterShowFriend](images/GUITestImages/AfterShowFriend.png) |
+
+As you can see in the above images `Bean` was deleted with the appropriate command result message displayed in the CommandResult box.
+
+If incorrect format of delete command is entered, the appropriate delete friend error message should be displayed in the CommandResult box. Shown below is the error message in the CommandResultBox.
+
+![ShowFriendError](images/GUITestImages/ShowFriendError.png)
+
+#### Show all friends
+
+Upon successful execution of `showfriends` command the following must occur
```

Comment:
just a note that we have to update the documentation from `showfriends` to `listfriends` in the next iteration (same for events)!

Reply:
Yes! I will update the GUI test documentation in the next iteration